packages/open-next/src/adapters/routing/middleware.ts

```diff
@@ -123,18 +123,13 @@ export async function handleMiddleware(
   // If the middleware returned a Redirect, we set the `Location` header with
   // the redirected url and end the response.
   if (res.statusCode >= 300 && res.statusCode < 400) {
-    const location = result.response.headers
-      .get("location")
-      ?.replace("http://localhost:3000", `https://${req.headers.host}`);
+    resHeaders.location = resHeaders.location?.replace("http://localhost:3000", `https://${req.headers.host}`);
```

Comment:
Not sure this will work on every case, what if you set the uppercase `Location` headers in the middleware ?

Reply:
here i mentioned that it doesn't matter uppercase or lowercase, key will be lowercase always
https://github.com/sst/open-next/issues/368#issuecomment-1964869309